src/reducers/membershipsReducer.js

```diff
@@ -0,0 +1,39 @@
+export default function(state = {}, action) {
+  switch (action.type) {
+    case 'POST_MEMBERSHIP_PENDING':
+      return state;
+
+    case 'POST_MEMBERSHIP_FULFILLED':
+      return Object.assign({}, state, {
+        [action.payload.pursuance_id]: action.payload
+      });
+
+    case 'POST_MEMBERSHIP_REJECTED':
+      return state;
+
+    case 'GET_MEMBERSHIPS_PENDING':
+      return state;
+
+    case 'GET_MEMBERSHIPS_FULFILLED':
+      return Object.assign({}, state, ...action.payload);
+
+    case 'GET_MEMBERSHIPS_REJECTED':
+      return state;
+
+    case 'DELETE_MEMBERSHIP_PENDING':
+      return state;
+
+    case 'DELETE_MEMBERSHIP_FULFILLED':
+      const {
+        [String(action.payload.pursuance_id)]: _,
```

Comment:
Interesting, what does the `_` do/mean?

Reply:
It signifies that the variable(Deleted membership) will not be used.